src/app/features/shared/components/atoms/ToggleCollapse/ToggleCollapse.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Button, themeSpacing } from "@datapunt/asc-ui"
+import { ButtonVariant } from "@datapunt/asc-ui/lib/components/Button/Button"
+import styled from "styled-components"
+
+type Props = Omit<React.ComponentProps<typeof Button>, "onClick"> & {
+  clickHandler: () => void
+  isCollapsed: boolean
+  displayInline? : boolean
+  buttonVariant?: ButtonVariant
```

Comment:
Maybe nicer to reuse ASC `Button` prop names. `onClick` and `variant`, instead of `clickHandler` and `buttonVariant`?